websites/G/GoPlay/metadata.json

```diff
@@ -2,18 +2,22 @@
 	"$schema": "https://schemas.premid.app/metadata/1.11",
 	"apiVersion": 1,
 	"author": {
-		"name": "Striker",
-		"id": "215509157837537280"
+		"name": "Frostware",
+		"id": "1268984565930786899"
 	},
-	"service": "Anontpp",
+	"service": "GoPlay",
 	"description": {
-		"en": "Anontpp is the home to the anime streaming service, GoPlay. You can watched free shows uninterupted in HD with no ads.",
-		"ga_IE": "Is é Anontpp an baile don tseirbhís sruthú anime, GoPlay. Is féidir leat féachaint ar thaispeántais saor in aisce gan bhriseadh i HD gan aon fhógraí.",
-		"nl": "Anontpp is de thuisbasis van de anime-streamingdienst GoPlay. U kunt ononderbroken gratis shows in HD bekijken zonder advertenties.",
-		"vi_VN": "Anontpp là nơi để xem anime trực tuyến, GoPlay. Bạn có thể xem các chương trình miễn phí, không bị gián đoạn bởi các quảng cáo."
+		"en": "Goplay is the home to the kdrama streaming service. You can watched free shows uninterupted in HD with no ads.",
+		"ga_IE": "Is é Goplay an baile don tseirbhís sruthú kdrama. Is féidir leat féachaint ar thaispeántais saor in aisce gan bhriseadh i HD gan aon fhógraí.",
+		"nl": "GoPlay is de thuisbasis van de anime-streamingdienst GoPlay. U kunt ononderbroken gratis shows in HD bekijken zonder advertenties.",
+		"vi_VN": "GoPlay là nơi để xem kdrama trực tuyến. Bạn có thể xem các chương trình miễn phí, không bị gián đoạn bởi các quảng cáo."
 	},
-	"url": "goplay.pw",
-	"version": "1.1.14",
+	"url": [
+                 "goplay.ml",
+                 "goplay.su",
+                 "kdrama.anontpp.com"
+	],
+	"version": "1.0.0",
 	"logo": "https://cdn.rcd.gg/PreMiD/websites/A/Anontpp/assets/logo.png",
 	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/A/Anontpp/assets/thumbnail.png",
```

Comment:
Since you are renaming the presence, please download and re-upload these images to allow the process to move the images to the correct URL if this gets merged.

Reply:
done